README.md

```diff
@@ -314,24 +313,33 @@ module.exports = {
 };
 ```
 
-### specs
-**Required.** The list of paths where `hermione` will look for tests.
+### sets
+You can link some set of tests with certain browsers using sets. If sets are not specified and paths
+to test are not passed from cli, hermione will look for test in `hermione` folder.
```

Comment:
... hermione will look for testS in `hermione` folder *and run them in all browsers*